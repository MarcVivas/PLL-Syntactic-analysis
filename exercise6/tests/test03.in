// Alphabet definition
ALPHABET => a, b <=

// States definition
STATES =>  ->q0, q1-> <=   // q0 is an initial state and q1 is an accepting state

// Transitions definitions
TRANSITIONS =>
->q0[a]...->q0;
->q0[b]...q1->;
q1->[a]...q1->;
q1->[b]...q1->;
<=
