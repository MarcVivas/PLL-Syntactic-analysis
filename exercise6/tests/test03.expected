Start states: { ->q0 }
Accepting states: { q1-> }
Intermediate states:
Alphabet: a b
Transitions:
{ ->q0 } ... { ->q0 } [symbol="a"];
{ ->q0 } ... { q1-> } [symbol="b"];
{ q1-> } ... { q1-> } [symbol="a"];
{ q1-> } ... { q1-> } [symbol="b"];

